files/packit-httpd.conf

```diff
@@ -0,0 +1,44 @@
+# MIT License
```

Comment:
Why have this here if it's already in [deployment](https://github.com/packit-service/deployment/blob/master/openshift/cmap-packit-httpd-conf.yml.j2)?

Reply:
b/c you literally can't run p-s locally without this file, so I will remove it from the deployment repo

hence better local development

Reply:
Removing this from the deployment repo will force us to rebuild the service image whenever we would like to tweak httpd-packit.conf in production. Is this what we want?

Reply:
I assume this change was required to have httpd-packit.conf available when running a local deployment with docker-compose. Local deployments done on ab `oc cluster up` local cluster, using the deployment playbooks already had this file mounted.